wxTextCtrl and and wxMenuItem updates

pstieber opened the pull request:
While attempting to build gdevelop on Fedora 21 using GCC 4.9.2 against a current wxWidgets master built with 2.8 compatibility turned off, I ran into a few compilation errors that were fixed with these changes.

4ian commented:
Looks good! One thing though, some of the code that have been edited were generated using a RAD tool called wxSmith (integrated into Code::Blocks). This way of making dialogs is deprecated (instead it's better to use wxCrafter for new dialogs) but it's maybe a good idea to keep wxSmith files in sync with the code generated (if someone want to edit the dialogs later with this tool): you should remove wxTE_AUTO_SCROLL in [this file](https://github.com/4ian/GD/blob/master/IDE/wxsmith/BugReport.wxs),  [here](https://github.com/4ian/GD/blob/master/IDE/wxsmith/MAJ.wxs#L73) and [here too](https://github.com/4ian/GD/blob/master/IDE/wxsmith/EventsEditor.wxs#L36).

If you can do it, I'll merge this PR shortly after :+1:

pstieber commented:
Thanks for the review.  I removed wxTE_AUTO_SCROLL from the files you listed.  I also found it in [ConsoleFrame.wxs](https://github.com/4ian/GD/blob/master/IDE/wxsmith/ConsoleFrame.wxs#L44), so I removed it there too.

4ian commented:
Looks great! :+1:  Thanks :)
